tests/api/test_generated_endpoints.py

```diff
@@ -94,6 +95,45 @@ def test_get_data(self, generator, model, route, controlled_values, client, Grap
         for value in controlled_values:
             assert actual['data']['attributes'][value] == expected[camelCase_to_underscore(value)]
 
+    def test_can_delete_work(self, client, normalized_data_id):
```

Comment:
Can you also add a test to make sure that deleted works are filtered out of list endpoints?